### PR TITLE
Make Microsoft.Subscription.Admin api consistent.

### PR DESCRIPTION
**Pull request by yusonMicrosoft**
* Remove location delete api

* Change Metrics to metrics

* Change MetricDefinitions to metricDefinitions

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by yusonMicrosoft — COMMENTED** [review]
@bganapa @deathly809 @martabarillas

**Review by yusonMicrosoft — COMMENTED** [review]
@bganapa @deathly809 @martabarillas

File: specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Location.json [hunk 1]
```diff
@@ -114,37 +114,6 @@
                         }
                     }
                 }
-            },
-            "delete": {
-                "x-ms-examples": {
-                    "Deletes the specified location.": {
-                        "$ref": "./examples/Location/Delete.json"
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Location.json]: can you pls remove the Delete.json in the examples

Comment (nschonni) [inline specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Location.json]: The unreferenced file is what is failing Avocado

**Review by veronicagg — CHANGES_REQUESTED** [review]
As @bganapa commented, please remove the example you're not consuming anymore. Other than that, looks good.

File: specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Location.json (hunk 1 above)
Comment (yusonMicrosoft) [reply specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Location.json]: It's done. Thanks.

**Review by bganapa — APPROVED** [review]

**Comment by bganapa** [comment]
@veronicagg This should be good to merge. There was no delete API on the location, API spec was having it by mistake.. hence the PR

**Review by veronicagg — APPROVED** [review]

**Pull request merged**